File: 黑马_新浪微博/day9/emoji/DemoViewController.m
//
//  DemoViewController.m
//  emoji
//
//  Created by 翟佳阳 on 2021/12/9.
//

#import "DemoViewController.h"
#import "emoji-Swift.h"
@interface DemoViewController ()

@end

@implementation DemoViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    NSString *str = @"0x1f609";
    NSLog(@"%@",[str emoji]);
    Test2 *test = [Test2 new];
    [test show];
    
    

}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: 黑马_新浪微博/day4/OC单例/DemoViewController.swift
//
//  DemoViewController.swift
//  OC单例
//
//  Created by 翟佳阳 on 2021/11/25.
//

import UIKit

class DemoViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
//        print(SoundTools.sharedTools3)
        // Do any additional setup after loading the view.
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
//        print(SoundTools.sharedTools4)
        print(NetWorkTools.shared())
        print("aaaaaaaa")
    }

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
